No error reporting for functional tests.

I want to create a very simple test, but it fails and I can't figure out why:

```lua
local helpers = require('test.functional.helpers')
local clear, feed, insert = helpers.clear, helpers.feed, helpers.insert
local execute, expect = helpers.execute, helpers.expect

describe('nvim', function()
  before_each(clear)

  describe('can insert NULs', function()
    feed('ia<C-V>000b<Esc>')
    expect('a\x00b')
  end)
end)
```

The output:

> ●●●●●●●●●●●●●
13 successes / 0 failures / 0 errors / 0 pending : 0.0 seconds
CMake Error at /home/admin/src/neovim/cmake/RunTests.cmake:22 (message):
  Unit tests failed.

If I delete the `can insert NULs` test, it succeeds (still 13 successes), but I have no idea why it fails.

<bountysource-plugin>

---
Want to back this issue? **[Place a bounty on it!](https://www.bountysource.com/issues/5285820-no-error-reporting-for-functional-tests?utm_campaign=plugin&utm_content=tracker%2F461131&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F461131&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/src/nvim/normal.c
+++ b/src/nvim/normal.c
@@ -1942,8 +1942,11 @@ void do_pending_operator(cmdarg_T *cap, int old_col, bool gui_yank)
          * the lines. */
         auto_format(false, true);
 
-        if (restart_edit == 0)
+        if (restart_edit == 0) {
           restart_edit = restart_edit_save;
+        } else {
+          cap->retval |= CA_COMMAND_BUSY;
+        }
       }
       break;
 

--- a/src/nvim/ops.c
+++ b/src/nvim/ops.c
@@ -2060,7 +2060,7 @@ void op_insert(oparg_T *oap, long count1)
   }
 
   t1 = oap->start;
-  edit(NUL, false, (linenr_T)count1);
+  (void)edit(NUL, false, (linenr_T)count1);
 
   // When a tab was inserted, and the characters in front of the tab
   // have been converted to a tab as well, the column of the cursor

--- a/src/nvim/testdir/test_visual.vim
+++ b/src/nvim/testdir/test_visual.vim
@@ -15,3 +15,16 @@ func Test_block_shift_multibyte()
   call assert_equal('	ヹxxx', getline(2))
   q!
 endfunc
+
+func Test_Visual_ctrl_o()
+  new
+  call setline(1, ['one', 'two', 'three'])
+  call cursor(1,2)
+  set noshowmode
+  set tw=0
+  call feedkeys("\<c-v>jjlIa\<c-\>\<c-o>:set tw=88\<cr>\<esc>", 'tx')
+  call assert_equal(['oane', 'tawo', 'tahree'], getline(1, 3))
+  call assert_equal(88, &tw)
+  set tw&
+  bw!
+endfu

--- a/src/nvim/version.c
+++ b/src/nvim/version.c
@@ -661,7 +661,7 @@ static const int included_patches[] = {
   // 294,
   // 293,
   // 292,
-  // 291,
+  291,
   290,
   // 289,
   // 288 NA
@@ -670,7 +670,7 @@ static const int included_patches[] = {
   // 285 NA
   // 284 NA
   // 283,
-  // 282,
+  282,
   // 281 NA
   280,
   // 279 NA

